src/sequences/__init__.py

```diff
@@ -63,6 +63,7 @@ def get_next_value(
     *,
     nowait=False,
     using=None,
+    increment=1,  # private API
```

Comment:
Wouldn't this lead to users also being able to use `get_next_value` as `get_next_values()`?

Reply:
Strictly speaking, yes, but it's undocumented and I don't think it will cause issues in practice.

The alternative would be to rename this function to `_get_next_value` and to redefine `get_next_value` as a wrapper adding the `increment=1` keyword argument. I feel like this would be performative :-)

Reply:
Upon further thought, with the rise in popularity of IDE (the most common editor is not Visual Studio Code), there's a fair chance that people could discover this argument because the editor "helpfully" suggests it. I'll add the wrapper.